package.json

```diff
@@ -65,5 +72,9 @@
     "react-test-renderer": "^16.13.1",
     "tern": "^0.24.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node-fetch": "^2.6.1",
+    "plist": ">=3.0.2",
```

Comment:
It seems to me like you can remove this just fine as well. There is no conflicting entries in `yarn.lock`.